### PR TITLE
[bitnami/*] Adapt values.yaml of Apache, ASP.NET Core and Cassandra to readme-generator

### PR DESCRIPTION
**Pull request by pablogalegoc**
**Description of the change**

Add metadata to `values.yaml` in order to be used by readme-generator to automatically generate the "Parameters" section of the `README.md`.

**Benefits**

Less manual work and consistency between `values.yaml` and `README.md`.

**Additional information**

Commented keys are not picked by readme-generator so some had to be uncommented in order to appear in the README, however none of them affect the default installation of the chart, verified comparing the output before and after the changes of
```bash
helm install chart . -f values.yaml --dry-run --debug
```

**Checklist** 

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])

### CODE REVIEW

File: bitnami/cassandra/values.yaml [hunk 1]
```diff
@@ -33,188 +69,56 @@ image:
...
-# existingConfiguration:
+dbUser:
+  user: cassandra
+  forcePassword: false
+  # password:
+
+  ## Use an existing secrets which already stores your password data.
+  ## for backwards compatibility, existingSecret can be a simple string,
+  ## referencing the secret by name.
+  # existingSecret:
```
Comment (alemorcuq) [inline bitnami/cassandra/values.yaml:93]: We are not adding some variables like `password` or `existingSecret` to the README. Are we ok with that?

File: bitnami/aspnet-core/README.md [hunk 2]
```diff
@@ -47,153 +47,148 @@ The command removes all the Kubernetes components associated with the chart and
...
+| `healthIngress.tls`             | Enable TLS configuration for the hostname defined at ingress.hostname parameter                            | `false`                  |
+| `healthIngress.secrets`         | If you're providing your own certificates, please use this to add the certificates as secrets              | `[]`                     |
+
 
 ### RBAC parameters
 
-| Parameter                               | Description                                                         | Default                                                 |
-|-----------------------------------------|---------------------------------------------------------------------|---------------------------------------------------------|
-| `serviceAccount.create`                 | Enable the creation of a ServiceAccount for ASP.NET Core pods       | `true`                                                  |
-| `serviceAccount.name`                   | Name of the created ServiceAccount                                  | Generated using the `aspnet-core.fullname` template     |
```
Comment (alemorcuq) [inline bitnami/aspnet-core/README.md:195]: We are losing these kind of comments saying that the default value is auto generated. Are we ok with that?

File: bitnami/cassandra/values.yaml (hunk 1 above)
Comment (pablogalegoc) [reply bitnami/cassandra/values.yaml]: Now they are added! Thanks

File: bitnami/aspnet-core/README.md (hunk 2 above)
Comment (pablogalegoc) [reply bitnami/aspnet-core/README.md:195]: I think it's ok as long a it is kept in the `values.yaml` like in here:https://github.com/bitnami/charts/blob/6b528440e4737010b736cdcaebbf241bcde2c0e4/bitnami/redis/values.yaml#L999-L1001

**Review by alemorcuq — APPROVED** [review]

**Pull request merged**